UM/VersionUpgradeManager.py

```diff
@@ -244,7 +244,6 @@ def _findShortestUpgradeRoutes(self) -> Dict[Tuple[str, int], Tuple[str, int, Ca
     #   \return The filename of each file relative to the specified directory.
     def _getFilesInDirectory(self, directory: str) -> Iterator[str]:
         for (path, directory_names, file_names) in os.walk(directory, topdown = True):
-            directory_names[:] = []  # Only go to one level.
```

Comment:
@diegopradogesto I think the reason this only goes one level deep is because of the `old` folders. I would suggest to avoid creating the `old` folders so we avoid further issues.

Reply:
Ah, `old` is already ignored. Then this is fine.

Reply:
Wait, now I just realize that the version upgrade is also analyzing all the files in the plugins folder, looking for version number of those files. That will take a lot of time if people have many plugins with lots of files.